CONTRIBUTING.md

```diff
@@ -3,6 +3,6 @@ Contributing
 
 Ace is a community project and wouldn't be what it is without contributions! We actively encourage and support contributions. The Ace source code is released under the BSD License. This license is very simple, and is friendly to all kinds of projects, whether open source or not. Take charge of your editor and add your favorite language highlighting and keybindings!
 
-Feel free to fork and improve/enhance Ace any way you want. If you feel that the editor or the Ace community will benefit from your changes, please open a pull request.
+Feel free to fork and improve/enhance Ace any way you want. If you feel that the editor or the Ace community will benefit from your changes, please open a pull request. And make sure that your changes are covered by test cases to pass the pull request checks which exists inorder to keep ace robust.
```

Comment:
nitpick: starting a sentence with `And` feels a bit off, maybe just do `Please make sure that your changes are covered by ...`

Reply:
ACK